src/main/java/br/unicamp/cecom/appointmentscheduler/core/features/admin/AdminRestController.java

```diff
@@ -0,0 +1,58 @@
+package br.unicamp.cecom.appointmentscheduler.core.features.admin;
+
+import br.unicamp.cecom.appointmentscheduler.core.features.admin.to.request.CreateAdminRequest;
+import br.unicamp.cecom.appointmentscheduler.core.features.admin.to.request.UpdateAdminRequest;
+import javassist.NotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.Optional;
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.created;
+import static org.springframework.http.ResponseEntity.noContent;
+
+@RestController
+@RequiredArgsConstructor
```

Comment:
Add autowired for dependency injection:
```suggestion
@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```